qa/tasks/cephfs/test_volumes.py

```diff
@@ -7888,3 +7888,46 @@ def test_unparseable_metafile_on_legacy_to_v1_upgrade(self):
 
         # remove group
         self._fs_cmd("subvolumegroup", "rm", self.volname, group)
+
+
+class TestEmptyStringForCreates(CephFSTestCase):
+    CLIENTS_REQUIRED = 1
+    MDSS_REQUIRED = 1
+
+    def _fs_cmd(self, *args):
+        return self.mgr_cluster.mon_manager.raw_cluster_cmd("fs", *args)
+
+    def setUp(self):
+        super().setUp()
+        result = json.loads(self._fs_cmd("volume", "ls"))
+        self.assertTrue(len(result) > 0)
+        self.volname = result[0]['name']
+
+    def tearDown(self):
+        # clean up
+        super().tearDown()
+
+    def test_empty_name_string_for_subvolumegroup_name(self):
+        """
+        To test that an empty string is unacceptable for a subvolumegroup name
+        """
+        with self.assertRaises(CommandFailedError):
+            self._fs_cmd("subvolumegroup", "create", self.volname, "''")
```

Comment:
nit: Wouldn't it be better if we check both -- empty argument as well empty quotes -- like the way we do in test method `test_empty_name_string_for_subvolumegroup_name_argument`?
```suggestion
        with self.assertRaises(CommandFailedError):
            self._fs_cmd("subvolumegroup", "create", self.volname, "''")
        with self.assertRaises(CommandFailedError):
            self._fs_cmd("subvolumegroup", "create", self.volname, "")            
```

Reply:
added the test case